app/Resources/views/base.html.twig

```diff
@@ -45,6 +46,22 @@
             <!-- Collect the nav links, forms, and other content for toggling -->
             <div class="collapse navbar-collapse navbar-ex1-collapse">
                 <ul class="nav navbar-nav side-nav">
+                    <li class="{%if dashboard is defined %} active {% endif%}">
```

Comment:
@GITFenix tutaj nie trzeba przekazywać zmiennych otóż twig w symfony przetrzymuje dla nas np request.

Więcej tutaj:

http://symfony.com/doc/current/reference/twig_reference.html#global-variables

Ogólnie można to zrobić w taki sposób:

```html
<li class="{% if app.request.attributes.get('_route') == 'dashboard' %}active{% endif %}">
     <a href="{{ path('dashboard') }}"><i class="fa fa-dashboard"></i> Dashboard</a>
</li>
```

Można też użyć "bundla" (czyli paczki) typu https://packagist.org/packages/knplabs/knp-menu-bundle ale to by było trochę na wyrost w tym wypadku ;)